src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs

```diff
@@ -189,6 +189,8 @@ public virtual AccessibleStates State
         /// </summary>
         public virtual AccessibleObject? GetChild(int index) => null;
 
+        internal virtual int GetChildIndex(AccessibleObject child) => -1;
```

Comment:
🤔  Can it be `null`?
```suggestion
        internal virtual int GetChildIndex(AccessibleObject? child) => -1;
```